uicomponent-compose/feed/src/main/java/io/github/droidkaigi/feeder/feed/FeedItem.kt

```diff
@@ -116,6 +117,15 @@ fun FeedItem(
                     if (feedItem is FeedItem.Podcast) onClickPlayPodcastButton(feedItem)
                 }
         )
+        AudioSpectrumAnimation(
+            isPlayingPodcast = isPlayingPodcast,
+            isVisible = feedItem is FeedItem.Podcast,
+            modifier = Modifier
+                .constrainAs(audioSpectrum) {
+                    linkTo(image.top, image.bottom, bias = 1.0f)
+                    linkTo(image.start, image.end, bias = 1.0f)
+                }
+        )
```

Comment:
How about changing visibility like this? If we have isVisible in the component, I think the component will be complicated. If you have any problems with ConstraintLayout, please let me know.

```kotliin
if (feedItem is FeedItem.Podcast) {
          AudioSpectrumAnimation(
            isPlayingPodcast = isPlayingPodcast,
            modifier = Modifier
                .constrainAs(audioSpectrum) {
                    linkTo(image.top, image.bottom, bias = 1.0f)
                    linkTo(image.start, image.end, bias = 1.0f)
                }
        )
}
```

Reply:
> If we have isVisible in the component, I think the component will be complicated.

Sounds good! 😄 I think so too, thank you! I also think this component does not need `isPlayingPodcast`, what did you think?

On a different note, I think the same applies to `AudioControlButton` component about `isVisibility`, what do you think?

Reply:
Sounds good! That looks better! 👍 
How about fixing only AudioSpectrumAnimation now?

Reply:
I'll fix it! 🙌🏻